engine/src/flutter/lib/ui/platform_dispatcher.dart

```diff
@@ -1823,6 +1880,10 @@ class _PlatformConfiguration {
     String? defaultRouteName,
     String? systemFontFamily,
     int? configurationId,
+    double? lineHeightScaleFactorOverride,
```

Comment:
Is the `copyWith` method used anywhere (it's a private class so I assume it is)? Just want to make sure nobody is using that to unset the overrides because you can't do that using `copyWith`.

Reply:
I think this should be alright `systemFontFamily` is also `nullable` and it is present in this `copyWith`. Generally I feel like if a class has a nullable member that it includes in its `copyWith` it should be okay if all the `copyWith` does it let's you set it to some non-null value. This is generally how i've seen this used in the framework, one example is `TextStyle`.